src/app/crypto-handler.ts

```diff
@@ -1,7 +1,7 @@
 import { app } from 'electron';
```

Comment:
we don't need native encryption anymore. So, we might as well remove this.

Reply:
Native encryption work if we disable `contextIsolation`